### DISCUSSION TITLE
angular/could-not-find-module-angular-devkit-build-angular/

### DISCUSSION THREAD

**Discussion by giscus** [arungudelli/angularjswiki#198] [Announcements]
# angular/could-not-find-module-angular-devkit-build-angular/

To fix Could not find module angular-devkit-build-angular error in Angular follow the below steps 1. Delete node_modules folder and run npm install 2. Install `angular-devkit-build-angular` package as a dev dependency.

https://www.angularjswiki.com/angular/could-not-find-module-angular-devkit-build-angular/

**Comment by ednsinf** [comment]
STUPID SOLUTION NOT WORKS FOR ME
"Node packages may not be installed. Try installing with 'npm install'.
Error: Could not find the '@angular-devkit/build-angular:dev-server' builder's node package."
